packages/terra-infinite-list/src/InfiniteList.jsx

```diff
@@ -0,0 +1,435 @@
+import React from 'react';
```

Comment:
Similar to the utils file, I think adding some more doc to this file would help make it easier to reason about and maintain in the future.

Reply:
Updated.